napari/layers/vectors/vectors.py

```diff
@@ -408,7 +408,7 @@ def _get_state(self):
                     else [self._edge.current_color]
                 ),
                 'edge_color_cycle': self.edge_color_cycle,
-                'edge_colormap': self.edge_colormap.dict(),
```

Comment:
Why this change? As far as I can tell, `edge_colormap` is a `Colormap` object. Or am I lost in the code?

Reply:
No, it's `tuple[str, Colormap]`, so to get the `Colormap` it needs indexing.

Reply:
Oh, but I think that's because the property was incorrectly typed - in the latest commit I've fixed the typing of `edge_colormap` and reverted the change on this line.